Texture.podspec

```diff
@@ -65,10 +62,4 @@ Pod::Spec.new do |spec|
   spec.default_subspec = 'PINRemoteImage'
 
   spec.social_media_url = 'https://twitter.com/TextureiOS'
-  spec.library = 'c++'
-  spec.pod_target_xcconfig = {
-       'CLANG_CXX_LANGUAGE_STANDARD' => 'c++11',
-       'CLANG_CXX_LIBRARY' => 'libc++'
-  }
-
```

Comment:
(See PR description). These options have been here since the initial OSS release and aren't adding value anymore. The default clang C++ options under Xcode 9 (and probably Xcode 8) are sufficient.